modules/atomic-widgets/prop-types/svg-prop-type.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropTypes;
+
+use Elementor\Modules\AtomicWidgets\Image_Sizes;
+use Elementor\Modules\AtomicWidgets\PropTypes\Base\Object_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Primitives\String_Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Svg_Prop_Type extends Object_Prop_Type {
+	public static function get_key(): string {
+		return 'image/svg+xml';
```

Comment:
This is the `key` for the prop type and not a mime type, maybe `svg` will fit better here.